regression in `getChainTips` test

<!-- 
Please use this template while reporting a bug and provide as much info as possible.

If the matter is security related, please disclose it privately via security@defichain.com
-->

#### What happened:

Test failing due to regression. 

Test CI Logs https://github.com/DeFiCh/jellyfish/pull/323/checks?check_run_id=2713213400

#### Test summary 

```javascript
Summary of all failing tests
FAIL packages/jellyfish-api-core/__tests__/category/blockchain/getChainTips.test.ts (9.21 s)
  ● ChainTips › should getChainTips

    expect(received).toBeGreaterThan(expected)

    Expected: > 0
    Received:   0

      20 |     for (let i = 0; i < chainTips.length; i += 1) {
      21 |       const data = chainTips[i]
    > 22 |       expect(data.height).toBeGreaterThan(0)
         |                           ^
      23 |       expect(typeof data.hash).toStrictEqual('string')
      24 |       expect(data.hash.length).toStrictEqual(64)
      25 |       expect(data.branchlen).toBeGreaterThanOrEqual(0)

      at __tests__/category/blockchain/getChainTips.test.ts:22:27
      at fulfilled (__tests__/category/blockchain/getChainTips.test.ts:5:58)
          at runMicrotasks (<anonymous>)


Test Suites: 1 failed, 138 passed, 139 total
Tests:       1 failed, 989 passed, 990 total
Snapshots:   0 total
Time:        669.067 s
Ran all test suites in 13 projects.
Force exiting Jest: Have you considered using `--detectOpenHandles` to detect async operations that kept running after all tests finished?
Error: Process completed with exit code 1.
```

--- a/packages/jellyfish-api-core/__tests__/category/blockchain/getChainTips.test.ts
+++ b/packages/jellyfish-api-core/__tests__/category/blockchain/getChainTips.test.ts
@@ -9,7 +9,7 @@ describe('ChainTips', () => {
   beforeAll(async () => {
     await container.start()
     await container.waitForReady()
-    await container.waitForWalletCoinbaseMaturity()
+    await container.waitForBlock(1)
   })
 
   afterAll(async () => {

--- a/packages/jellyfish-api-core/__tests__/category/blockchain/getChainTips.test.ts
+++ b/packages/jellyfish-api-core/__tests__/category/blockchain/getChainTips.test.ts
@@ -9,6 +9,7 @@ describe('ChainTips', () => {
   beforeAll(async () => {
     await container.start()
     await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
   })
 
   afterAll(async () => {

